stratum/hal/lib/phal/db.proto

```diff
@@ -111,6 +112,18 @@ message ThermalGroup {
   repeated Thermal thermals = 1;
 }
 
+message OpticalTransceiver {
+  int32 id = 1;
+  string vendor_name = 2;
+  HwState hardware_state = 3;
+  double tx_power = 4;
```

Comment:
Why we have this `tx_power` field? 
I can't find anyone using this

Reply:
Updated. Please review.